### ISSUE TITLE
QR Code Certificate Download location not correct or not available

### ISSUE THREAD

**Issue by PeterQFR** [FreeTAKTeam/FreeTakServer#542]
I'm trying to download a certificate package via QR Code however my TAK Client - iTAK is failing to find it.

When I download my QR Code and decode it, it is pointing to:
```
http://192.168.1.30:8080/Marti/api/sync/metadata/6d6ae6955bb938bf43497c29423d00e5a6063460fd650cf5f833da8e7a12f080/tool
```
and when I try and go to that location it gives a 500 error. 192.168.1.30 is the correct computer.

**Comment by SCWhite** [comment]
yes, I'm having the same issue.
seems to have something to do with:
https://github.com/FreeTAKTeam/FreeTakServer/blob/296a6152b5c5a8ea41e41e984b4f4c9ffdbddd63/FreeTAKServer/core/services/RestAPI.py#L600

for some reason changing config file didn't have a effect on it

**Comment by brothercorvo** [comment]
we have now fixed the QR code issue #542, tested with an iPHONE (argh!)  the patch  will be released this WE
